pom.xml

```diff
@@ -82,11 +82,14 @@
     </developers>
 
     <properties>
-        <caf.boilerplate.service.version>2.2.0-SNAPSHOT</caf.boilerplate.service.version>
-        <caf.classification.service.version>1.1.0-SNAPSHOT</caf.classification.service.version>
+        <caf.boilerplate.service.version>2.1.0-48</caf.boilerplate.service.version>
+        <caf.classification.service.version>1.0.0-2</caf.classification.service.version>
```

Comment:
This should be on the 1.1.0 release of classification which is in progress. If it uses older version it loses the classification target = ALL work.